content/includes/header.html

```diff
@@ -11,33 +11,25 @@
 <meta name="HandheldFriendly" content="True">
 <meta name="MobileOptimized" content="320">
 <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
 <meta content="Vets.gov" property="og:site_name">
-
+{% if lastupdate %}
+<meta content="{{lastupdate}}" property="http://purl.org/dc/terms/date">
```

Comment:
I think the format for this is supposed to be `<meta content="{{lastupdate}}" name="DC.Date" >`.